app/controllers/notes_controller.rb

```diff
@@ -206,6 +206,12 @@ def preview
     render template: 'notes/show'
   end
 
+  def lock_note
```

Comment:
Here, i wonder if we could simply use the existing new tag route at tag_controller.rb, instead of a custom route. What do you think?